network/shuffle/pom.xml

```diff
@@ -43,6 +43,17 @@
       <version>${project.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.fusesource.leveldbjni</groupId>
+      <artifactId>leveldbjni-all</artifactId>
+      <version>1.8</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
```

Comment:
Maybe add the `jackson-annotation` module too, since you're using it? maven can get confused if transitive dependencies have a different version of these modules.